src/jaxsim/mujoco/__main__.py

```diff
@@ -0,0 +1,191 @@
+import argparse
+import pathlib
+import time
+
```

Comment:
```suggestion
import sys

```
Related to next suggestion

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/83/commits/4dcd4a760ac2d9afe7d6a74fea385f2447ebb07f